From c4a748deecb31b032c0c31a0a1e357565a0575f4 Mon Sep 17 00:00:00 2001
From: Aleksandr Slobodeniuk <aslobodeniuk@fluendo.com>
Date: Tue, 20 Jul 2021 01:55:39 +0200
Subject: [PATCH] fix build on Ubuntu focal

---
 config-custom.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config-custom.h b/config-custom.h
index 137b51f9..506a0b6b 100644
--- a/config-custom.h
+++ b/config-custom.h
@@ -40,3 +40,7 @@
 #undef near
 #undef far
 #endif /* _WIN32 */
+
+#ifndef STBI_SIMD
+#define STBI_SIMD 0
+#endif
-- 
2.25.1

